eng/pipelines/runtime.yml

```diff
@@ -393,6 +393,28 @@ jobs:
       - Browser_wasm
     alwaysRun: ${{ variables.isRollingBuild }}
 
+#
+# Temporarily add a leg that makes sure the wasm runtime with threading enabled
+# builds. We are working on making this a normal part of the wasm build.
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    buildConfig: release
+    runtimeFlavor: mono
+    platforms:
+      - Browser_wasm
+    jobParameters:
+      buildArgs: -s mono+libs -c $(_BuildConfig) /p:WasmEnableThreads=true
+      nameSuffix: Threading_AllSubsets_Mono
+      timeoutInMinutes: 180
+      condition: >-
+        or(
+          eq(dependencies.evaluate_paths.outputs['SetPathVars_libraries.containsChange'], true),
+          eq(dependencies.evaluate_paths.outputs['SetPathVars_mono.containsChange'], true),
+          eq(dependencies.evaluate_paths.outputs['SetPathVars_installer.containsChange'], true),
+          eq(variables['isRollingBuild'], true))
```

Comment:
`runtime-wasm` is only for manual runs, and runs all the jobs unconditionally. So, including it there would mean that we can run it on PRs without having to worry whether the yml would trigger the threaded build or not. So, I'm suggesting moving it to a separate template, and then using that in both runtime.yml, and runtime-extra-platforms-wasm.yml(only in the PR section) .

Reply:
@steveisok @radical I tried to do this, but quickly realized I have no idea what I'm doing. Specifically I'm confused by `condition:` and `- ${{ if eq(parameters.isRollingBuild, true) }}:` sections of some of these yml files.
> So, I'm suggesting moving it to a separate template, and then using that in both runtime.yml, and runtime-extra-platforms-wasm.yml(only in the PR section) .

Also: can we also add a second variant that uses `/p:WasmEnablePerfTracing=true` instead of `/p:WasmEnableThreads=true` so that we build both of the new configurations